server/services/inductionService.test.ts

```diff
@@ -9,26 +9,28 @@ import { aLongQuestionSetUpdateInductionRequest } from '../testsupport/updateInd
 import toWorkAndInterests from '../data/mappers/workAndInterestMapper'
 import toEducationAndTraining from '../data/mappers/educationAndTrainingMapper'
 import toInductionDto from '../data/mappers/inductionDtoMapper'
+import toCreateInductionRequest from '../data/mappers/createInductionMapper'
 import toUpdateInductionRequest from '../data/mappers/updateInductionMapper'
 import { aLongQuestionSetUpdateInductionDto } from '../testsupport/updateInductionDtoTestDataBuilder'
+import { aLongQuestionSetCreateInductionDto } from '../testsupport/createInductionDtoTestDataBuilder'
+import { aLongQuestionSetCreateInductionRequest } from '../testsupport/createInductionRequestTestDataBuilder'
 
+jest.mock('../data/educationAndWorkPlanClient')
 jest.mock('../data/mappers/workAndInterestMapper')
 jest.mock('../data/mappers/educationAndTrainingMapper')
 jest.mock('../data/mappers/inductionDtoMapper')
 jest.mock('../data/mappers/updateInductionMapper')
+jest.mock('../data/mappers/createInductionMapper')
 
 describe('inductionService', () => {
   const mockedWorkAndInterestsMapper = toWorkAndInterests as jest.MockedFunction<typeof toWorkAndInterests>
   const mockedEducationAndTrainingMapper = toEducationAndTraining as jest.MockedFunction<typeof toEducationAndTraining>
   const mockedInductionDtoMapper = toInductionDto as jest.MockedFunction<typeof toInductionDto>
   const mockedUpdateInductionMapper = toUpdateInductionRequest as jest.MockedFunction<typeof toUpdateInductionRequest>
+  const mockedCreateInductionMapper = toCreateInductionRequest as jest.MockedFunction<typeof toCreateInductionRequest>
 
-  const educationAndWorkPlanClient = {
-    getInduction: jest.fn(),
-    updateInduction: jest.fn(),
-  }
-
-  const inductionService = new InductionService(educationAndWorkPlanClient as unknown as EducationAndWorkPlanClient)
+  const educationAndWorkPlanClient = new EducationAndWorkPlanClient() as jest.Mocked<EducationAndWorkPlanClient>
```

Comment:
Changed this to use this better pattern that you introduced me to the other week @leejacobson-moj 👍